tox.ini

```diff
@@ -2,6 +2,7 @@
 project_name = uharfbuzz
 envlist = py3,py3-cov,coverage
 skip_missing_interpreters = true
+isolated_build = true
```

Comment:
I think I didn't add that on purpose back in the days. I don't want you to build the package itself because I am running setup.py develop myself so that some environment variable are set and enable some coverage tracing.

Reply:
I meant "tox", not "you" - misstyping from mobile phone

Reply:
Try to see if you can disable isolated build at least when tox env contains the "cov" particle (e.g. py310-cov):


isolated_build =
    cov: false
    !cov: true

I'm not sure if that is going to work but that was my intention

Reply:
That doesn't seem to work. I wonder if coverage etsting can be done more "cleanly"...

Reply:
Then just disable it for now

Reply:
The isolated build I mean